### ISSUE TITLE
Mod Suggestions?

### ISSUE THREAD

**Issue by Gpeschke** [EzioTheDeadPoet/SME-FT-#9]
It was mostly overlap between the modlist I was putting together and yours. Shows how much it's needed.
If you are interested, here's what I had that didn't overlap:

Corrupted NIFs Patches
Expression Override Function Fix
Missing Encounter Zones FIXED -
Private profile redirector
Unofficial Material Fix
Whiterun texture memory usage reduction

unofficial performance optimized textures- debatable include? yeah, most modders will probably grab a texture pack they like that'll override them, but for those who aren't a texture modder or who only grab piecemeal, it's a good non-altering perf boost.

I'm sure you looked into it, but I believe wiseman's floral fix was more compatible than floral respawn and fixed a few other plant related issues? Not entirely sure: https://www.nexusmods.com/skyrim/articles/50547

**Comment by EzioTheDeadPoet** [comment]
The Profile Redirector made it in, but the others are only really needed in connection with other mods that in my opinion are very taste dependent. And the fixed textures I am very undecided on because they make the installation time longer since most of them are still pretty large and imo they are to large of a download just for being replaced by other textures by most users.

**Comment by EzioTheDeadPoet** [comment]
- Due to it being buggy with ENBs and weather mods I decided to remove Private Profile Redirector again.

- I switched the Flora Fix out with a more user friendly and faster one.

- Missing Encounter Zones fixes one specific mod hence not included.

- NIF Patches again are patches for specific mods at least what I found on the Nexus.

- The Unofficial Materials Fix needs Static Mesh Improvement Mod as a requirement which I haven't included, cause it can cause incompatibilities with some texture mods.

- Whiterun Texture memory reduction didn't bring up anything on the nexus for me.

- unofficial performance textures and unofficial audio improvements aren't included to keep the size of the mods that need to be downloaded low and to focus on it being a pure base line. I could have put them in the optional mods section, but they wouldn't still would have needed to be downloaded and make the installation take longer for all the people not using it. All optional mods I included are small.
